src/index.js

```diff
@@ -1,27 +1,71 @@
 import './style.css';
-import Task from './modules/task.js';
 import MethodsTasks from './modules/methods.js';
 import { descriptFocusIn, descriptFocusOut } from './modules/interact.js';
 
 const methods = new MethodsTasks();
 const tasksList = document.querySelector('.tasks');
+const newDescription = document.getElementById('i--1');
+const submitBtn = document.getElementById('a--1');
 
-let todo = new Task('bench press', false, 0);
-methods.addTask(todo);
-todo = new Task('jump 20 times', false, 2);
-methods.addTask(todo);
-todo = new Task('jog for 20 minutes', true, 1);
-methods.addTask(todo);
+if (localStorage.getItem('tasks')) methods.tasks = JSON.parse(localStorage.getItem('tasks'));
 
 methods.resetList(tasksList);
 
-const checkboxes = document.querySelectorAll('.checkbox');
-const textareas = document.querySelectorAll('.description');
+newDescription.addEventListener('keydown', (e) => {
+  const description = newDescription.value;
+  e.preventDefault();
+  if (description.length > 0 && e.code === 'Enter') {
+    methods.addTask(description, false, methods.tasks.length);
+    methods.resetList(tasksList);
+    localStorage.setItem('tasks', JSON.stringify(methods.tasks));
+    newDescription.value = '';
+  }
+});
```

Comment:
- [ ] When adding tasks, I realized that the indexes of the tasks in the local storage are not updating correctly. There are two "index: 1" in the local storage which is not the desired behavior.

**This is an example of what I mean:**

![image](https://user-images.githubusercontent.com/86318284/184371804-78222dc8-a608-4fc9-87e6-af31165c432e.png)

Reply:
Thanks a lot, fixed the bugs and waiting for another review